lib/iris/tests/results/netcdf/netcdf_merc.cml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" ?>
+<cubes xmlns="urn:x-iris:cubeml-0.2">
+  <cube long_name="toa_brightness_temperature" standard_name="toa_brightness_temperature" units="K" var_name="data">
+    <attributes>
+      <attribute name="Conventions" value="CF-1.5"/>
+      <attribute name="Note" value="This dataset is for test purposes only"/>
+      <attribute name="acknowledgement" value="EUMETSAT"/>
+      <attribute name="creator_email" value="sat_systems@metoffice.gov.uk"/>
+      <attribute name="creator_name" value="Satellite Applications, Met Office"/>
+      <attribute name="creator_type" value="group"/>
+      <attribute name="geospatial_lat_max" value="0.0"/>
+      <attribute name="geospatial_lat_min" value="0.0"/>
+      <attribute name="geospatial_lon_max" value="0.0"/>
+      <attribute name="geospatial_lon_min" value="0.0"/>
+      <attribute name="history" value="Created: 2016-05-23T09:40:00Z"/>
```

Comment:
This is the result of a load test, so the attribute doesn't change; it is loaded in from a file.